src/main/java/org/jenkinsci/plugins/configfiles/folder/FolderConfigFileAction.java

```diff
@@ -96,7 +108,7 @@ public ContentType getContentTypeForProvider(String providerId) {
 
     @Override
     public HttpResponse doSaveConfig(StaplerRequest req) throws IOException, ServletException {
-        checkPermission(Job.CONFIGURE);
+        checkPermission(Item.CONFIGURE);
```

Comment:
unrelated - but it is actually defined on Item no Job...

Reply:
Ideally file this as a separate PR in general, this + the reorganized imports would have made a nice candidate for this, to keep only the gist here and make this more quickly reviewed/mergeable IMO.

Reply:
Yeah - but I blame eclipse for sorting the stuff when the new import was added.  (I did keep the removing unused imports in a different commit at least :-o )

Reply:
Eclipse, contrary to IDEA, actually has a view for staging chunks selectively, which anyone should always do with Git :).
Or use `git gui` ugly but functional OOTB tool.

Reply:
I [probably need to start using that again - but I tend to code and then build using command line and then commit in the command line cos well Jenkins extensions and eclipse just gives wired errors :-/

Will leave this to the maintaner as their choice if they want me to rewrite history or leave it as is.